tests/store/tracking/test_sqlalchemy_store.py

```diff
@@ -2057,8 +2075,13 @@ def test_search_with_max_results(store: SqlAlchemyStore):
     for n in [1, 2, 4, 8, 10, 20, 50, 100, 500, 1000, 1200, 2000]:
         assert runs[: min(1200, n)] == _search_runs(store, exp, max_results=n)
 
-    with pytest.raises(MlflowException, match=r"Invalid value for request parameter max_results"):
-        _search_runs(store, exp, max_results=int(1e10))
+    maxPlusOne = SEARCH_MAX_RESULTS_THRESHOLD + 1
```

Comment:
Actual change: update the test slightly. Still proving the same point, but compatible with the Go implementation now.